src/ts/components/subtitleoverlay.ts

```diff
@@ -505,8 +516,9 @@ export class SubtitleRegionContainerManager {
   }
 }
 
-class SubtitleRegionContainer extends Container<ContainerConfig> {
+export class SubtitleRegionContainer extends Container<ContainerConfig> {
   private labelCount = 0;
```

Comment:
I guess the `labelCount` can be removed now

Reply:
There is a public method using the count to remove the region container itself if there are no more labels inside